client/package.json

```diff
@@ -25,6 +25,7 @@
 		"@automattic/react-virtualized": "9.21.2",
 		"@automattic/tree-select": "file:../packages/tree-select",
 		"@babel/runtime": "7.8.3",
+		"@automattic/calypso-analytics": "file:../packages/calypso-analytics",
```

Comment:
Let's add the dependency to the root `package.json`, too (and keep it in `client/package.json`). That's needed to ensure reliable NPM installs and lockfiles. See #39218 (invented and merged yesterday) for more details.

Reply:
ok wow that's good to know I've made that change